spynl/main/mail.py

```diff
@@ -11,8 +11,8 @@
 from jinja2 import Environment, Template, TemplateNotFound, FileSystemLoader
```

Comment:
I think someone who knows what the deal is with dummy mailer should have a look here. Not sure what the intended behavior is.